torch/csrc/jit/codegen/cuda/index_compute.cpp

```diff
@@ -79,7 +79,10 @@ class ContigIDs : public OptInDispatch {
     if (!(std::all_of(
             ordered_inputs.begin(),
             ordered_inputs.end(),
-            [this](IterDomain* id) { return is_contig_root.at(id); }))) {
+            [this](IterDomain* id) {
+              return is_contig_root.at(id) && !id->isBroadcast() &&
+                  !id->isReduction();
```

Comment:
Does this actually fix the issue with broadcasting & contiguous simultaneously marked on a tensor?
The issue I reported in #230 actually went away in this PR.